electrumsv/network.py

```diff
@@ -1226,11 +1226,12 @@ def _random_server_nowait(self, protocol):
     async def _monitor_active_keys(self, account) -> None:
         '''Raises: RPCError, TaskTimeout'''
         all_keys = set(account.existing_active_keys())
+        session = await self._main_session()
+        monitored_keyinstance_ids = set([v[0] for k, v in session._keyinstance_map.items()])
```

Comment:
This should probably be `set([v[0] for v in session._keyinstance_map.values()])` just to be pendantic.

Reply:
Sure okay :)